packages/host/src/Instance.ts

```diff
@@ -14,6 +14,12 @@ import { exportData } from "./export";
 import type Host from "./Host";
 import BaseInstancePlugin from "./BaseInstancePlugin";
 
+const scriptCommands = [
+	"/cheat", "/editor",
```

Comment:
I don't think it is supported, my expectation is it would give an error but not until after it has disabled achievements. I also wanted to keep alignment with what the wiki calls "script commands" which /editor is a part of.